python-sdk/src/astro/sql/__init__.py

```diff
@@ -6,5 +8,18 @@
 from astro.sql.operators.load_file import LoadFileOperator, load_file
 from astro.sql.operators.merge import MergeOperator, merge
 from astro.sql.operators.raw_sql import RawSQLOperator, run_raw_sql
+from astro.sql.operators.run_query import RunQueryOperator
 from astro.sql.operators.transform import TransformOperator, transform, transform_file
 from astro.sql.table import Metadata, Table
+
+
+def get_value_list(sql_statement: str, conn_id: str, **kwargs) -> XComArg:
+    """
+    Execute a sql statement and return the result
+
+    :param sql_statement: sql query to execute
+    :param conn_id: Airflow connection id
```

Comment:
Same as https://github.com/astronomer/astro-sdk/pull/673/files#r962154439

Reply:
Improved this param description please let me know it looks now